src/json-editor.component.ts

```diff
@@ -175,6 +180,20 @@ export class JsonEditorComponent extends AbstractTrackerComponent implements OnI
     }
   }
 
+  onFieldAdd(field: string) {
+    this.keys = this.keys.add(field);
+    let tabName = this.schemaKeyToTabName[field];
```

Comment:
this should be only done when there are tabs, because it actually crashes when you add a key without `tabConfig`